doc/tools/apigen.py

```diff
@@ -457,9 +457,24 @@ def write_index(self, outdir, froot='gen', relative_to=None):
         w = idx.write
         w('.. AUTO-GENERATED FILE -- DO NOT EDIT!\n\n')
 
-        title = "API Reference"
+        title = "API Reference for scikit-image version 0.12.3"
```

Comment:
This version needs to change dynamically depending on the version of the docs built.